heartbeat/aws-vpc-move-ip

```diff
@@ -240,11 +238,21 @@ ec2ip_monitor() {
 		for rtb in $(echo $OCF_RESKEY_routing_table | sed -e 's/,/ /g'); do
 			ocf_log info "monitor: check routing table (API call) - $rtb"
 			if [ -z "${OCF_RESKEY_routing_table_role}" ]; then
-				cmd="$OCF_RESKEY_awscli --profile $OCF_RESKEY_profile $region_opt --output text ec2 describe-route-tables --route-table-ids $rtb --query RouteTables[*].Routes[?DestinationCidrBlock=='$OCF_RESKEY_ip/32'].$OCF_RESKEY_lookup_type"
+					if [[ "$OCF_RESKEY_ip" == *[:]* ]];
```

Comment:
You should use case to avoid these bashables.

Example:
https://github.com/ClusterLabs/resource-agents/blob/34e75a57b0aff462b5963d9d908bd1778ce5fdb1/heartbeat/exportfs#L283-L284

And also bump the indentation back a tab for this section.